phys2bids/physio_obj.py

```diff
@@ -542,6 +551,39 @@ def print_info(self, filename):
 
         LGR.info(info)
 
+    def auto_trigger_selection(self):
+        """
+        Find a trigger index matching the channels with a regular expresion.
+
+        Parameters
+        ----------
+        self
+         Raises
+        ------
+        Exception
+        More than one possible trigger channel was automatically found.
+        Exception
+        No trigger channel automatically found
+        Notes
+        -----
+        Outcome:
+            trigger_idx : int
+                Automatically retrieved trigger index
+        """
+        LGR.warning('Running automatic trigger detection.')
```

Comment:
I don't think this should be a warning but rather an `INFO`. phys2bids will have already warned the user after the parser. Now, I see this function as a feature in phys2bids, not as a function that tries to fix an issue or a bad argument the user has introduced.